app/src/main/java/com/infomaniak/drive/ui/fileList/FileAdapter.kt

```diff
@@ -71,6 +71,8 @@ open class FileAdapter(
 
     fun getFiles() = fileList
 
+    fun getValidItemSelected() = if (itemSelected.isManaged) itemSelected.filter { it.isValid } else itemSelected
```

Comment:
Can you add the return type to this function ?
It would help to understand what we are talking about here.